src/kd/systimer/systimer.c

```diff
@@ -416,6 +431,7 @@ s32 sceSTimerResetCount(s32 timerId)
     return SCE_ERROR_OK;
 }
 
+// 0664
```

Comment:
How about adding the hex prefix `0x` here so that this value number cannot mistakenly be interpreted as an address in decimal representation? Context-wise, it will probably be recognized as a hexadecimal representation, though we could use a preceeding `0x` to make it abundantly clear.

Personally, I always use the prefix `0x` for hex values to avoid any reason for ambiguity.

Reply:
Sure I can add it, we probably have a mix now across repo.

Reply:
Changed in 6512b5eaa7641c11c6f2e99d793f6dac0b39312c.